cli/CHANGELOG.md

```diff
@@ -3,11 +3,22 @@
 
 _Released 12/3/2024 (PENDING)_
 
+**Bugfixes:**
+
+- The CSS pseudo-class `:dir()` is now supported when testing in Electron. Addresses [#29766](https://github.com/cypress-io/cypress/issues/29766).
```

Comment:
@ryanthemanuel This should be moved below breaking changes actually

Reply:
Fixed: [`56524b7` (#30394)](https://github.com/cypress-io/cypress/pull/30394/commits/56524b7de45f13ec82da008e1814e414250ff684)